index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio: setup and mobile version skeleton</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+    <main class="container">        
+        <header>
```

Comment:
![port-1](https://user-images.githubusercontent.com/49184579/176822859-d4302d60-58ab-4c20-8742-de490e67632a.png)

Please compare your design and the figma and see the difference.
1. There is a margin around your content. Try if applying margin of 0 to body remove the space
2. You have set the width to 375px in container class and this is preventing your website from being responsive. You have to set the max-width to 100% or vw
3. The background image is not the same as in figma design
4. The font of your main title is not the same as in the figma design.  Everywhere its imperative that you follow the figma design for fonts, font-size, and color. The margin/padding should feel similar to the figma design.